app/design/frontend/rwd/default/template/page/html/header.phtml

```diff
@@ -15,6 +15,8 @@
 /**
  * @var Mage_Page_Block_Html_Header $this
  */
+$_currentStore = Mage::app()->getStore();
+$custom_header_image_option = Mage::getStoreConfig('design/header/header_image_option', $_currentStore);
```

Comment:
Can this be moved to the headers Block class (adding a new method?)

(and/or change to camelCase - for phpmd?)

Reply:
Instinctively, I think there is a way to not touch the phtml files at all. So that if this feature is merged, there is no need to change the custom phtml files in order to enjoy the new feature.